data/language/en-GB.txt

```diff
@@ -2895,7 +2895,7 @@ STR_5554    :{SMALLFONT}{BLACK}Enable mountain tool
 STR_5555    :Show vehicles from other track types
 STR_5556    :{SMALLFONT}{BLACK}Kick player
 STR_5557    :Stay connected after desynchronisation (Multiplayer)
-STR_5558    :A restart is required for this setting to take effect
+STR_5558    :This setting will only take effect after you have restarted OpenRCT2
```

Comment:
```suggestion
STR_5558    :This setting will only take effect after restarting OpenRCT2
```

Not sure which one I prefer. Initially I had thought of just adding a "game", so it would become "A game restart", but I guess any of our two suggestions are fine

Reply:
I wrote "you have restarted" to emphasise that it's the user's responsibility - OpenRCT2 won't restart by itself.

Reply:
Fine by me